### PR TITLE
feat: splash screen

### PR DESCRIPTION
**Pull request by limpbrains**
https://user-images.githubusercontent.com/155891/183834952-f72242a5-aefe-4d22-a87e-c3bc53cf57a7.mp4

I wanted to do this without adding extra dependencies, so I've followed [this tutorial](https://medium.com/@rfontes/splash-screens-on-react-native-without-libs-b487d08264ee). For iOS it is quite straightforward, but for android it requires additional code to hide loading screen.

in dev it still shows white screen when react is loading, should be fine in prod builds

### CODE REVIEW

**Comment by pwltr** [comment]
On Android there is a bit of black space in the top left corner.

https://user-images.githubusercontent.com/8538369/183963647-bc0b7fc6-99bc-4e45-8021-c87811ea28d6.mov

I wasn't able to get the splash screen on iOS (probably be a cache issue).

**Comment by MiguelMedeiros** [comment]
Yes, got the same here. 
Blank space on the left, also scrolling a little bit.

https://user-images.githubusercontent.com/5798170/184047477-a080ab81-e315-4bf6-8b19-afacd9211cfd.mp4

File: android/app/src/main/res/drawable/launch_screen.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:opacity="opaque">
+    <item android:drawable="@color/black" />
+    <item>
+        <bitmap android:src="@drawable/splash" android:gravity="center|clip_horizontal" />
```
Comment (Jasonvdb) [inline android/app/src/main/res/drawable/launch_screen.xml:7]: I think if you change `clip_horizontal` to `fill_horizontal` that might do the trick. Or one of [these](https://developer.android.com/reference/android/widget/LinearLayout.LayoutParams.html#attr_android:layout_gravity).

**Review by Jasonvdb — CHANGES_REQUESTED** [review]

**Comment by limpbrains** [comment]
Do you css is hard? try center-crop image on Android

**Review by Jasonvdb — APPROVED** [review]

**Pull request merged**